synced_versions_formulae.json

```diff
@@ -31,7 +31,6 @@
   ["docker", "docker-completion"],
   ["easyrpg-player", "liblcf"],
   ["etl", "synfig"],
-  ["file-formula", "libmagic"],
```

Comment:
These probably should remain synced, since `file-formula` depends on `libmagic`.

Reply:
yeah, that is true. let me revert the sync part